fcm-client.cabal

```diff
@@ -16,6 +16,8 @@ extra-doc-files:     README.md
 tested-with:         GHC == 8.8.4
                      GHC == 8.10.7
                      GHC == 9.0.2
+                     GHC == 9.2.5
+                     GHC == 9.4.2
```

Comment:
Latest available release in 9.4 is 9.4.4 so maybe it's worth bumping to that instead:
https://github.com/haskell/actions/tree/main/setup#version-support